share/spice/drupal/drupal.js

```diff
@@ -0,0 +1,78 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_drupal = function(api_result){
+
+        // Validate the response
+        if (!api_result || api_result.list === "") {
+            return Spice.failed('drupal');
+        }
+
+        // Render the response
+        Spice.add({
+            id: "drupal",
+            name: "Software",
+            data: api_result.list[0],
+            meta: {
+                sourceName: "Drupal.org",
+                sourceUrl: 'https://www.drupal.org/project/' + api_result.list[0].field_project_machine_name 
+            },
+            normalize: function(item) {
+                var boxData = [{heading: 'Project Information:'}];
+
+                if (item.field_project_homepage.url) {
```

Comment:
Actually, the default blank value is ```"field_project_homepage":{"url":""}```, so I have to check if item.field_project_homepage.url exists